packages/app/ui/modal/ModalHeaderTitle.tsx

```diff
@@ -9,17 +7,16 @@ interface ModalHeaderTitleProps {
 
 export const ModalHeaderTitle = ({
   title,
-  closeAction,
+  closeAction
 }: ModalHeaderTitleProps) => (
-  <ModalHeader>
-    <div className="flex items-center justify-between w-full">
-      <TitleText>{title}</TitleText>
-      <Button
-        action="quaternary"
-        iconLeft="close"
-        size="icon"
-        onClick={closeAction}
-      />
-    </div>
+  <ModalHeader className="flex items-center justify-between w-full pl-6 pr-4 pt-4 pb-2">
+    <TitleText size={2}>{title}</TitleText>
+    <Button
+      className="m-2.5 p-0"
```

Comment:
why?

I would keep the button as it is for now, for simplicity, and we can see if we need a custom style in the future.

Also don't think this style is better than the current one. But we can tweak it in the future if we feel the need.

Reply:
Awesome, will revert this `className` change then. It was purely out of replicating what I saw in the designs but I may be missing the mark too. Reverting!

Reply:
Not missing the mark at all. We do want people to explore and be audacious. Even if it means doing things worse every now and than. That's how sometimes something extremely good happens as well.